__tests__/endpoints.test.js

```diff
@@ -118,6 +157,26 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 
+  test('STATUS: 200, Returns single article object with all columns from articles table AND comment_count', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: 'Living in the shadow of a great man',
+            topic: 'mitch',
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 100,
+            comment_count: '11',
```

Comment:
this is the only new bit to the object, no? Only need to test the property I reckon